frontend_build/src/webpack.config.base.js

```diff
@@ -30,89 +30,106 @@ require('module').Module._initPaths();
 
 var fs = require('fs');
 var webpack = require('webpack');
-var jeet = require('jeet');
 var merge = require('webpack-merge');
 
-var aliases = require('./apiSpecExportTools').coreAliases();
-
-var postCSSPlugins = function () {
-  return [require('autoprefixer')];
-};
-
 var production = process.env.NODE_ENV === 'production';
 var lint = (process.env.LINT || production);
 
-aliases['kolibri_module']= path.resolve('kolibri/core/assets/src/kolibri_module');
+var aliases = require('./apiSpecExportTools').coreAliases();
+aliases['kolibri_module'] = path.resolve('kolibri/core/assets/src/kolibri_module');
 aliases['content_renderer_module'] = path.resolve('kolibri/core/assets/src/content_renderer_module');
+aliases['keen_ui_variables'] = path.resolve('kolibri/core/assets/src/keen-config/variables.scss');
+
 
-require('./htmlhint_custom'); // adds custom rules
+// helps convert to older string syntax for vue-loader
+var combineLoaders = require('webpack-combine-loaders');
 
+// for stylus blocks in vue files
+const vueStylusLoaders = [
+  'vue-style-loader', // includes postcss processing
+  {
+    loader: 'css-loader',
+    options: { minimize: production, sourceMap: !production }
+  },
+  'stylus-loader'
+];
+
+if (lint) {
+  vueStylusLoaders.push('stylint-loader')
+}
+
+// for scss blocks in vue files (e.g. Keen-UI files)
+const vueSassLoaders = [
+  'vue-style-loader', // includes postcss processing
```

Comment:
yeah, I like the consistency of all objects, also. can change that back. Maybe make them one-liners